### DISCUSSION TITLE
qbittorrent errors out when permissions set as default

### DISCUSSION THREAD

**Discussion by jbaquero05** [Luctia/ezarr#40] [Help wanted]
Hello,

I've been working on setting up a server where Jellyseerr sends requests to Prowlarr, which in turn communicates with Radarr, and finally, the torrent is sent to qBittorrent using ezarr. Everything seems to be connecting properly, but I'm encountering an issue during the download stage.

The error message I'm getting is:

`
File error alert. Torrent: "movie.file". File: "/data/media/movie.file". Reason: "movie.file file_open (/data/media/movie.file) error: Permission denied"
`

This setup is on a fresh Debian installation using regular Docker and Portainer for convenience. I haven't made any changes to file structures, folder names, users, permissions or anything from what Ezarr initially configures.

I'm unsure if this is a folder mapping problem or if there's a specific user permission I need to set that I'm not aware of. I've searched online for information about this error but haven't found anything relevant, making me think I might be missing something fundamental.

I've sought help on Discord servers, and the suggestion is to stop using Ezarr and follow the TRaSH guides. However, due to my limited familiarity with Linux and Docker, that's not a viable option for me.

Any assistance or insights would be greatly appreciated.

[This](https://privatebin.net/?8cada01a8ca15696#BM1Ff44S9Brwd6G3BXGRMfrtaBrjAdoYk7mXxKD883z
) is a compose of my entire docker setup if anyone wants to take a look. 

Thanks!

**Comment by Luctia** [comment]
Could you share your docker compose again? The privatebin link has expired.
